### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'push')

### ISSUE THREAD

**Issue by MagePsycho** [rostilos/perspective-partytown#1]
Hello Team,

I installed the extension and configured the settings. 
But it's not working for me. It's giving the below error:
> TypeError: Cannot read properties of undefined (reading 'push')
>    at partytown-ww-sw.js?v=0.7.6:1791:79
>    at receiveMessageFromSandboxToWorker

Do you guys have proper guidelines?

**Comment by rostilos** [comment]
Hi, @MagePsycho 

Could you please clarify what version of M2 and php you are using, and also about the configurations. Also : do you use https?

I would like to point out that the module is currently in alpha stage, and there may be minor issues with php8.1 support as well

**Comment by MagePsycho** [comment]
@rostilos 
I am using Magento version `2.4.6-p4` and PHP version `8.2`. 
I am currently testing locally with the warden and the test domain. is: `https://mage246.test` 


And the settings look like this:
![PartyTown Settings](https://github.com/rostilos/perspective-partytown/assets/1205071/f406a7ca-3af6-423a-9dd0-f67b30916df8)

Let me know if anything is needed.

**Comment by rostilos** [comment]
Added a new version of the module ( it is still in alpha stage ) but has php8 support and the latest version of partytown lib npm packages. Installation instructions are in the project's Readme.md.
I tested on php8.2 + m2 2.4.6-p4 using warden and reverse-proxy on nodejs. Please try installing the new version. Thanks in advance for your reply

**Comment by MagePsycho** [comment]
@rostilos 
I am getting errors while running the composer commands:
```
composer config repositories.perspective-magento2-partytown git git@github.com:rostilos/perspective-partytown.git
composer require perspective/magento2-partytown dev-master
```
> [Composer\Repository\InvalidRepositoryException]
>  No valid composer.json was found in any branch or tag of git@github.com:rostilos/perspective-partytown.git, could not load a package from it.

**Comment by rostilos** [comment]
Yes, the composer.json file was in the wrong directory for this. Fixed this and some other bugs as well

**Comment by MagePsycho** [comment]
@rostilos 
Thanks for the quick fix. I got the installation via composer working fine.
Trying to set up the reverse proxy in warden. Let me know if you have any docs for it.

Thanks

**Comment by rostilos** [comment]
There is an example implementation on nodejs
https://github.com/rostilos/perspective-partytown/blob/master/docs/reverse-proxy.md

**Comment by rostilos** [comment]
@MagePsycho 

Added a sample reverse-proxy implementation for warden ( with the addition of a new container with nginx as reverse-proxy )

https://github.com/rostilos/perspective-partytown/blob/master/docs/reverse-proxy-nginx.md
